### PR TITLE
docs: Add information about coding style

### PR DESCRIPTION
**Pull request by dumitrache-adrian92**
Add a paragraph that explains what coding style should be used and how to use checkpatch.pl to find style errors.

This will make it easier for people to start contributing.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -20,3 +20,4 @@ Please be civil when taking part in discussions.
 For pull requests, please follow the [GitHub flow](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork): create a fork of the repository, create your own branch, do commits, push changes to your branch, do a pull request (PR).
 The destination branch of pull requests is the default `master` branch.
 
+For code contributions, please respect the [Linux Kernel coding style](https://www.kernel.org/doc/html/v4.10/process/coding-style.html). To make this process easier, download [checkpatch.pl](https://github.com/systems-cs-pub-ro/iocla/blob/master/scripts/checkpatch.pl) and run [this command](https://github.com/systems-cs-pub-ro/iocla/blob/master/.github/workflows/main.yml#L12), it will show you all style errors. Alternatively, there's a [VS Code extension](https://marketplace.visualstudio.com/items?itemName=idanp.checkpatch) that integrates the patch into its editor (follow the instructions on the extension page).
```
Comment (teodutu) [inline CONTRIBUTING.md:23]: Place each sentence in this paragraph on a separate line. This way it'll be easier to review. You should add [this paragraph](https://github.com/open-education-hub/operating-systems/blob/master/CONTRIBUTING.md#content-writing-style) to `CONTRIBUTING.md` as well.

**Review by teodutu — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -20,3 +20,15 @@ Please be civil when taking part in discussions.
 For pull requests, please follow the [GitHub flow](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork): create a fork of the repository, create your own branch, do commits, push changes to your branch, do a pull request (PR).
 The destination branch of pull requests is the default `master` branch.
 
+For code contributions, please respect the [Linux Kernel coding style](https://www.kernel.org/doc/html/v4.10/process/coding-style.html).
```
Comment (teodutu) [inline CONTRIBUTING.md:23]:
```suggestion
## Coding Style

For code contributions, please respect the [Linux Kernel coding style](https://www.kernel.org/doc/html/v4.10/process/coding-style.html).
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -20,3 +20,15 @@ Please be civil when taking part in discussions.
...
+To make this process easier, download [checkpatch.pl](https://github.com/systems-cs-pub-ro/iocla/blob/master/scripts/checkpatch.pl) and run [this command](https://github.com/systems-cs-pub-ro/iocla/blob/master/.github/workflows/main.yml#L12), it will show you all style errors.
+Alternatively, there's a [VS Code extension](https://marketplace.visualstudio.com/items?itemName=idanp.checkpatch) that integrates the patch into its editor (follow the instructions on the extension page).
+
+For contributions to the documentation, write each sentence on a new line.
```
Comment (teodutu) [inline CONTRIBUTING.md:27]:
```suggestion
## Content Writing Style

For contributions to the documentation, write each sentence on a new line.
```

**Review by teodutu — CHANGES_REQUESTED** [review]
The extra headings make the document easier to read and it looks less like a wall of text.

**Comment by dumitrache-adrian92** [comment]
Ok, my mistake. I wanted to make a separate pull request with the previous commit for a bunch of coding style changes. It went automatically here however. How do I go about fixing this?

**Comment by teodutu** [comment]
> Ok, my mistake. I wanted to make a separate pull request with the previous commit for a bunch of coding style changes. It went automatically here however. How do I go about fixing this?

It would be easier if you had made this PR from a branch other than master. Now given the circumstances you should do the following:

```
$ git checkout -b fix-coding-style  # Create another branch from master. It will contain all these commits at first.

$ git reset HEAD~3 --hard  # Remove the last 3 commits (those in this PR) and their code changes.

$ cd /root/of/this/repo  # For the next command to work.

$ git checkout master -- laborator  # Get the changes to the laboratory/ folder from master on the new branch

$ git commit -sm 'your commit message'

$ git push -u origin fix-coding-style  # Push the "clean" coding style fixes onto another remote branch. Make a PR from this one.

$ git checkout master  # Start cleaning up the master branch.

$ git reset HEAD^ --hard  # Remove the last commit (the one with the code changes).

$ git push -f  # Remove it from the remote history as well.
```

**Comment by teodutu** [comment]
You can use `git log` between either of the steps I suggested to check your current commits.

**Comment by dumitrache-adrian92** [comment]
That worked, thank you.

**Review by teodutu — APPROVED** [review]

**Pull request merged**